src/vivarium_census_prl_synth_pop/data/loader.py

```diff
@@ -112,12 +112,19 @@ def load_persons(key: str, location: str) -> pd.DataFrame:
     data.relation_to_household_head = data.relation_to_household_head.map(
         metadata.RELATIONSHIP_TO_HOUSEHOLD_HEAD_MAP
     )
-
-    # put all non-draw columns in the index, else vivarium will drop them
-    data = data.set_index(
-        ["census_household_id", "age", "relation_to_household_head", "sex", "race_ethnicity"]
+    # Map native born persons and if person has migrated in last year
+    data["born_in_us"] = data["born_in_us"].map(
+        metadata.NATIVITY_MAP
+    )  # True for native citizens
+    data.loc[
+        data["immigrated_in_last_year"].isnull(), "immigrated_in_last_year"
+    ] = 1.0  # Make Nulls map to False
```

Comment:
It might be cleaner to do this after the `MIGRATION_MAP` is applied.

Reply:
You mean make them False here after the map is applied?